en/docs/Learn/APIMonetization/monetizing-an-api.md

```diff
@@ -0,0 +1,703 @@
+# Monetizing an API
+
+API Monetization is the process that allows API publishers to earn money by charging developers for using their services that have been exposed as APIs. When using API Monetization, API publishers can select billing plans based on their requirements and API providers are able to maximize their revenue by selecting the most profitable pricing plan for their APIs.
```

Comment:
Should we reword this sentence to maybe "API Monetization allows an organization to grow their business and generate revenue by providing services or exposing data via APIs".We can then maybe go into details in explaining who an API Publisher is ?

Reply:
"When using API Monetization, API publishers can select billing plans based on their requirements and API providers are able to maximize their revenue by selecting the most profitable pricing plan for their APIs"  - This sentence is not very clear. Can we re think this?

Reply:
Fixed